nemo/collections/asr/parts/submodules/rnnt_decoding.py

```diff
@@ -208,15 +208,17 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
         self.compute_timestamps = self.cfg.get('compute_timestamps', None)
         self.word_seperator = self.cfg.get('word_seperator', ' ')
 
-        if self.durations is not None:  # this means it's a TDT model.
+        if self.durations is not None and self.durations != []:  # this means it's a TDT model.
```

Comment:
I personally prefer checking for bool, which is more idiomatic and follows PEP8 
> For sequences (strings, lists, tuples), use the fact that empty sequences are false
https://peps.python.org/pep-0008/#programming-recommendations

But the current approach is also acceptable for me since it works.